init/src/synthpop.proto

```diff
@@ -0,0 +1,110 @@
+syntax = "proto3";
+package synthpop;
+
+message Population {
+    repeated Household households = 1;
+    repeated Person people = 2;
+
+    // This is filled out per stringified Activity. The exception is for
+    // `Activity::Home`; see `households` for that.
+    map<string, VenueList> venues_per_activity = 3;
+
+    map<string, InfoPerMSOA> info_per_msoa = 4;
+}
+
+message Household {
+    uint64 id = 1;
+    string msoa = 2;
+    /// An ID from the original data, kept around for debugging
+    int64 orig_hid = 3;
+    // These IDs index into `Population::people`
+    // TODO These're redundant; worth putting here?
+    repeated uint64 members = 4;
+}
+
+message VenueList {
+    repeated Venue venues = 1;
+}
+
+message InfoPerMSOA {
+    repeated Point shape = 1;
+    uint64 population = 2;
+    // All building centroids within this MSOA. May be empty.
+    repeated Point buildings = 3;
+}
+
+// In WGS84
+message Point {
+    float longitude = 1;
+    float latitude = 2;
+}
+
+message Person {
+    uint64 id = 1;
+    uint64 household = 2;
+    // This is the centroid of the household's MSOA. It's redundant to store it per person, but
+    // very convenient.
+    Point location = 3;
+    // An ID from the original data, kept around for debugging
+    int64 orig_pid = 4;
+    // The Standard Industry Classification for where this person works. 0 means none.
+    uint64 sic1d07 = 5;
+
+    uint32 age_years = 6;
+
+    Health health = 7;
+
+    repeated Flows flows_per_activity = 8;
+}
+
+message Health {
+    Obesity obesity = 1;
+    // TODO Unclear what the values mean
```

Comment:
@HSalat you sent a doc that had lots of details about all of these, but it's buried in Slack somewhere... Was it a google doc or overleaf, do you remember?

Reply:
it's on overleaf. Here's. a copy:

idp: individual identifier as MSOA11CD_XXXXXX, where XXXXXX is the rank when the data are sorted by increasing hid, then decreasing age
MSOA11CD: Middle-layer Super Output Area as defined for the 2011 census where the individual lives
hid: household identifier, includes communal establishments

Identifiers for other data sets:
pid: 2011 Census
pid_tus: Time Use Survey 2015
pid_hse: Health Survey for England 2017

General information:
sex: 0 female, 1 male
age: in years
origin: 1 White, 2 Black, 3 Asian, 4 Mixed, 5 Other

Working and social characteristics:
nssec5: National Statistics Socio-economic classification:
    1. Higher managerial, administrative and professional occupations
    2. Intermediate occupations
    3. Small employers and own account workers
    4. Lower supervisory and technical occupations
    5. Semi-routine and routine occupations
    0. Never worked and long-term unemployed
soc2010: Previous version of the Standard Occupational Classification -- see: https://www.ons.gov.uk/methodology/classificationsandstandards/standardoccupationalclassificationsoc/soc2010
sic1d07: Standard Industrial Classification of Economic Activities 2007, 1st layer (number corresponding to the letter in alphabetical order) -- see: https://www.ons.gov.uk/methodology/classificationsandstandards/ukstandardindustrialclassificationofeconomicactivities
sic2d07: Standard Industrial Classification of Economic Activities 2007, 2nd layer -- see: https://www.ons.gov.uk/methodology/classificationsandstandards/ukstandardindustrialclassificationofeconomicactivities

Proportion of 24h spent doing different daily activities:
punknown + pwork + pschool + pshop + pservices + pleisure + pescort + ptransport  = pnothome
phome + pworkhome = phometot
pnothome + phometot = 1

Lockdown 'essential worker' status -- source: London in Lockdown: Batty et al. https://arxiv.org/abs/2011.07165:
keyworkercasa: -> we should change that to bool

Health data:
cvd: has a cardio-vascular disease (0 or 1)
diabetes: has diabetes (0 or 1)
bloodpressure: has high blood pressure (0 or 1)
BMIvg6: Body Mass Index:
    Not applicable
    Underweight: less than 18.5
    Normal: 18.5 to less than 25
    Overweight: 25 to less than 30
    Obese I: 30 to less than 35
    Obese II: 35 to less than 40
    Obese III: 40 or more
BMI_healthier: same as above, but everyone from Overweight to Obese III has been moved 1 rank higher in the above list (healthier population assumption)

Likelihoods compared to other individuals (i.e. not directly a probability) -- source: see Events WIP paper:
ESport: attending a live sports event
ERubgy: attending a live rugby event
EConcertF: attending a live concert primarily targeting a young female audience
EConcertM: attending a live concert primarily targeting a young male audience
EConcertFS: attending a live concert primarily targeting a middle-aged female audience
EConcertMS: attending a live concert primarily targeting a middle-aged male audience
EMuseum: visiting a museum
Geolocation columns:
lng: Longitude of the MSOA11CD centroid
lat:: Latitude of the MSOA11CD centroid